lib/api/CDataFrameAnalyzer.cc

```diff
@@ -35,9 +35,16 @@ const char FINISHED_DATA_CONTROL_MESSAGE_FIELD_VALUE{'$'};
 
 // Result types
 const std::string ROW_RESULTS{"row_results"};
-const std::string PROGRESS_PERCENT{"progress_percent"};
 const std::string ANALYZER_STATE{"analyzer_state"};
 
+// Phase progress
+const std::string PHASE_PROGRESS{"phase_progress"};
+const std::string PHASE{"phase"};
+const std::string PROGRESS_PERCENT{"progress_percent"};
+
+// Temporarily the only progress phase
```

Comment:
How about just...
```suggestion
// Progress phases
```
then the comment won't immediately go out-of-date.